Dockerfile

```diff
@@ -1,16 +1,26 @@
 FROM ruby:2.6.7-buster
 
-# docker-ce-cli apt dependencies
 ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update
 
-# TODO: Add python stuff
+# Install python
+RUN apt-get install -y python3 python3-pip
 
-# The working location in the container is:
-WORKDIR /app
+# Install python/pip
+# RUN python3 -m ensurepip
+RUN pip3 install --no-cache --upgrade pip setuptools
+
+# Get speaker-verification repo
+RUN git clone https://github.com/OnTrack-UG-Squad/speaker-verification.git
```

Comment:
do you need to install any dependencies for this?

Reply:
Yes, I'll need to update it to make the virtual environment and install the requirements.txt stuff, as per the main speaker-verification readme instructions.

https://github.com/OnTrack-UG-Squad/speaker-verification

Reply:
Rather than going about the virtual environment stuff, we've just gone ahead and installed the dependencies.  The docker container doesn't really need things to be kept in a virtual environment from what I can see.

We have since run into problems on the speaker-verification side of things.  It appears some packages have since been updated and are now causing problems which we will have to investigate.